packages/dd-trace/src/tracer.js

```diff
@@ -122,6 +122,33 @@ class DatadogTracer extends Tracer {
 <meta name="dd-trace-id" content="${traceId}" />\
 <meta name="dd-trace-time" content="${traceTime}" />`
   }
+
+  /**
+   * Links an authenticated user to the current trace
+   * @param {object} user Properties of the authenticated user. Accepts custom fields
+   * @param {string} user.id Unique identifier of the user. Mandatory
+   * @param {string} [user.email] Email of the user
+   * @param {string} [user.name] User-friendly name of the user
+   * @param {string} [user.session_id] Session ID of the user
+   * @param {string} [user.role]
+   * @param {string} [user.scope]
+   * @returns {boolean} If the operation succeeded
```

Comment:
Is this used anywhere? This is not something that is common in the API of the library.

Reply:
It's mostly used in unit tests to make sure we've branched off the nominal case tbh. But I also feel weird to just silently fail when a span is not found for example. If that's ok with you I can change it to return nothing.